linetools/isgm/utils.py

```diff
@@ -527,6 +547,15 @@ def group_coincident_compoments(comp_list, output_type='list'):
     elif output_type in ['dict', 'dictionary']:
         return output_dict
 
+def wherein(groups,member):
+    ''' Once blends have been identified, find which blend a given line index belongs to.
```

Comment:
Please add documentation regarding input parameters, and what this function will return. Could you also name this function something more meaningful for the general user?